docs/application/web/api/5.5/device_api/wearable/tizen/cordova/media.html

```diff
@@ -146,11 +146,17 @@ <h2>1. Interfaces</h2>
 <div class="constructors">
 <h4 id="Media::constructor">Constructors</h4>
 <dl>
+<dt class="method" id="">
+<a class="backward-compatibility-anchor" name=""></a><code><b><span class="methodName">Constructor (DOMString, <a href="cordova.html#SuccessCallback">SuccessCallback</a>?, <a href="#MediaErrorCallback">MediaErrorCallback</a>?, <a href="#StatusChangeCallback">StatusChangeCallback</a>?)</span></b></code>
+</dt>
 <div class="brief">
  The Media interface provides interface and functions to manage audio objects.
               </div>
 <pre class="webidl prettyprint">Media(DOMString src, optional <a href="cordova.html#SuccessCallback">SuccessCallback</a>? successCallback, optional <a href="#MediaErrorCallback">MediaErrorCallback</a>? errorCallback,
       optional <a href="#StatusChangeCallback">StatusChangeCallback</a>? mediaStatus);</pre>
+<p><span class="version">Since: </span>
```

Comment:
Each API element can be introduced since some version. Line 157 is related to Constructor. Line 193 is 'since' related to Constants section. In this conditions, the behaviour is correct.